lib/ex.coffee

```diff
@@ -32,21 +32,19 @@ trySave = (func) ->
 
   deferred.promise
 
-saveAs = (filePath) ->
-  editor = atom.workspace.getActiveTextEditor()
-  fs.writeFileSync(filePath, editor.getText())
```

Comment:
This function's disappearance has me worried (I missed the call removals just as well as the length cases disappearing): it fixed stuff because Atom's API wasn't low-level enough.